examples/python/utils_hd.py

```diff
@@ -0,0 +1,53 @@
+from typing import Dict, Tuple
+
+from eth_account.messages import encode_typed_data, SignableMessage
+from ledgereth.accounts import find_account
+from ledgereth.comms import init_dongle
+from ledgereth.messages import sign_typed_data_draft
+from starknet_py.net.signer.stark_curve_signer import KeyPair
+
+from utils import (
+    build_stark_key_message,
+    get_acc_contract_address_and_call_data,
+    get_private_key_from_eth_signature
+)
+
+
+def sign_stark_key_message_ledger(message: SignableMessage, eth_account_address: str) -> str:
+    dongle = init_dongle()
+    account = find_account(eth_account_address, dongle, count=1)
```

Comment:
I would suggest `count=10` at least since we have multiple addresses in our hardware wallet and the Paradex address is not the first one, took me awhile to figure this out!